healthcare/api-client/v1/datasets/datasets.py

```diff
@@ -0,0 +1,413 @@
+# Copyright 2020 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+
+from googleapiclient import discovery
+from googleapiclient.errors import HttpError
+
+
+# [START healthcare_get_client]
+def get_client():
+    """Returns an authorized API client by discovering the Healthcare API and
+    creating a service object using the service account credentials in the
+    GOOGLE_APPLICATION_CREDENTIALS environment variable."""
+    api_version = 'v1'
+    service_name = 'healthcare'
+
+    return discovery.build(
+        service_name,
+        api_version)
+# [END healthcare_get_client]
+
+
+# [START healthcare_create_dataset]
+def create_dataset(
+        project_id,
+        cloud_region,
+        dataset_id):
```

Comment:
Does this function fit within 79 chars? If so why not having this in one line?

Reply:
Done.